Test/COD145904.TXT

```diff
@@ -0,0 +1,186 @@
+OBJECT Codeunit 145904 Test_DotNet_Regex_Toolkit
+{
+  OBJECT-PROPERTIES
+  {
+    Date=;
+    Time=;
+    Version List=;
+  }
+  PROPERTIES
+  {
+    OnRun=BEGIN
+          END;
+
+  }
+  CODE
+  {
+    VAR
+      TxtError@1000000000 : TextConst 'ENU=Arguments: %1\Result: %2\Expected: %3';
+    PROCEDURE VerifyBooleanResult@1000000020(Arguments@1000000003 : Text;Result@1000000001 : Boolean;ExpectedResult@1000000000 : Boolean);
+    BEGIN
+      VerifyResult(Arguments, Result = ExpectedResult, Result, ExpectedResult);
+    END;
+    PROCEDURE VerifyIntegerResult@1000000004(Arguments@1000000003 : Text;Result@1000000001 : Integer;ExpectedResult@1000000000 : Integer);
+    BEGIN
+      VerifyResult(Arguments, Result = ExpectedResult, Result, ExpectedResult);
+    END;
+    PROCEDURE VerifyTextResult@1000000006(Arguments@1000000003 : Text;Result@1000000001 : Text;ExpectedResult@1000000000 : Text);
+    BEGIN
+      VerifyResult(Arguments, Result = ExpectedResult, Result, ExpectedResult);
+    END;
+    PROCEDURE VerifyResult@1000000003(Arguments@1000000003 : Text;Verified@1000000001 : Boolean;Result@1000000000 : Variant;ExpectedResult@1000000004 : Variant);
+    BEGIN
+      IF ( NOT (Verified)) THEN
+        ERROR(TxtError, Arguments, Result, ExpectedResult);
+    END;
+    PROCEDURE ToString1@1000000007(Value1@1000000000 : Variant) : Text;
```

Comment:
This function only adds boilerplate.

Just use `STRSUBSTNO` with whatever number of arguments you want 😃